src/app/views/censusdata/create.html

```diff
@@ -72,7 +51,7 @@
                 <label class="control-label col-md-3">Weather:</label>
                 <div class="col-md-9">
                 <div class="checkbox">
-                    <label ng-repeat="weather in data.weatherInfo"><input type="radio" name="weatherInfo" ng-model="censusForm.weatherInfo" value="{{weather.Id}}"> {{ weather.Description }} &nbsp;</label>      
+                    <label ng-repeat="weather in data.weatherInfo"><input type="radio" name="weatherInfo" ng-model="censusForm.weatherInfoId" value="{{weather.Id}}"> {{ weather.Description }} &nbsp;</label>      
```

Comment:
Weather should be a radio button instead of checkboxes. We also need to add two text boxes: Temperature (F) and Average Wind Speed. Both will be integer values. I need to add those to the data model, as I missed them previously.